README.md

```diff
@@ -1,3 +1,11 @@
 # Conways-Game-Of-Life
 
 An implemention in Rust of the [Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life) made by John Conway. This is work in progress.
+
+To run the game, use:
+
+`cargo run start 1y 1x 2y 2x ... Ny Nx`
```

Comment:
```suggestion
`cargo run start 1x 1y 2x 2y ... Ny Nx`
```
In your code x goes first
Also, I think that a little explanation of parameters' meaning would be useful for newcomers, just say that those are the coordinates of alive cells.

Reply:
Yes, I had a confusion about this because the traditional x-y axes are rotated in the drawing. For the UI version I will clarify which is X and which is Y.